app.js

```diff
@@ -0,0 +1,6279 @@
+function drawing(size) {
+    let field = document.querySelector('canvas'),
+        ctx = field.getContext('2d'),
+        width = square[0].length,
+        height = square.length,
```

Comment:
width and height don't change, why do you calculate it? I think it should be belong in the upper scope.

Reply:
fixed